src/features/persons/button_actions/useButtonActions.tsx

```diff
@@ -191,13 +191,14 @@ const useButtonActions = () => {
       }
 
       if (dataView !== 'main') {
-        const viewExist =
-          person.person_data.categories.value.includes(dataView);
+        const categories = Array.from(
```

Comment:
I wanna wrap these func inside a callback but I guess we can dedicate a task to work on performance improv and take care of that?